metadata.yaml

```
@@ -72,6 +72,12 @@ requires:
     limit: 1
     description: |
       Send a CSR to- and obtain a signed certificate from an external CA.
+  forward-auth:
+    interface: forward_auth
+    limit: 1
+    description: |
+      Receives forwardAuth middleware config from Oathkeeper.
+      The same middleware will be applied to all charms that requested Identity and Access Proxy protection.
```

Just updating with my understanding, which could be wrong!
```suggestion
      Receive config from e.g. oathkeeper, for rendering the forwardAuth middleware.
      The same middleware is applied to all proxied endpoints that requested Identity and Access Proxy (IAP) protection.
      For this reason we set a relation count limit of 1.
```